src/ray/gcs/gcs_server/test/gcs_job_manager_test.cc

```diff
@@ -102,14 +102,6 @@ TEST_F(GcsJobManagerTest, TestGetJobConfig) {
       *add_job_request2,
       &empty_reply,
       [](Status, std::function<void()>, std::function<void()>) {});
-
-  auto job_config1 = gcs_job_manager.GetJobConfig(job_id1);
-  ASSERT_EQ("namespace_1", job_config1->ray_namespace());
```

Comment:
I don't think you should delete the namespace check.

Reply:
My bad. It was unintentionally introduced as well. reverted.